cronjobs/cnn_classify_data.py

```diff
@@ -72,6 +76,15 @@
                         help='str array with columns obsdate, tilenumber, separated by |')
 
     args = parser.parse_args()
+
+    
+    #DESITRIP_daily
+#     logic = getattr(sys.modules[__name__], args.logic)
+
+    tile_numbers = args.tilenum
+    obsdate = args.obsdate
```

Comment:
Are these lines needed? It doesn't seem like they are being used in the code

Reply:
Yeah, we could do away with them, but I use them inside the for loop.

Reply:
If these lines are not needed they should be removed.

Reply:
These lines are required because they are the variable describing the tile number and observation date.